experimental/ubuntu/julia-0.6.4/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Start with DBR python virtual env image
+FROM databricksruntime/python-virtualenv:latest
```

Comment:
is it required to start from the python base?

Reply:
What is in the python-virtualenv base image? If this is a standard set of libraries we use on databricks, then I think these will be preferred for a data scientist.

Reply:
just python and some basic packages